src/halmos/solvers.py

```diff
@@ -307,8 +368,12 @@ def install_solver(solver: SolverInfo) -> Path:
         if install_path.exists():
             raise RuntimeError(f"File already exists: {install_path}")
 
+        ui.print(
+            f"Solver saved to [bold]{install_path}[/bold] ([cyan]{format_size(len(content))}[/cyan])",
+            highlight=False,
+            markup=True,
+        )
```

Comment:
experimenting with some richer output, looks like this

<img width="819" alt="image" src="https://github.com/user-attachments/assets/22290907-aa70-4f90-abfb-251d139a5008" />